timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -566,12 +575,18 @@ private static boolean canEvaluateExpression(UExpression expression) {
   }
 
   private static boolean isCallFromMethodInSubclassOf(JavaContext context, UCallExpression call,
-      String methodName, String className) {
+      String methodName, Class type) {
     JavaEvaluator evaluator = context.getEvaluator();
     PsiMethod method = call.resolve();
     return method != null //
         && methodName.equals(call.getMethodName()) //
-        && evaluator.isMemberInSubClassOf(method, className, false);
+        && evaluator.isMemberInSubClassOf(method, type.getCanonicalName(), false);
+  }
+
+  private static boolean isCallFromMethodInSubclassOf(JavaContext context,
+          KotlinUQualifiedReferenceExpression expression, String methodName, Class type) {
+    return isSubclassOf(context, expression.getReceiver(), type)
+            && expression.getSelector().asSourceString().equals(methodName);
```

Comment:
Ugh.  This is brittle, but currently Lint doesn't have a better way of doing this, so should be fine for now.  I'll file an issue and update.